pkg/controller/template/render_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 var (
-	updateGoldenFiles = flag.Bool("u", false, "If set to True, the tests will update the golden files before testing.")
+	updateGoldenFiles = flag.Bool("u", true, "If set to True, the tests will update the golden files before testing.")
```

Comment:
why changing this to `true`? during normal unit tests we don't want to regenerate

Reply:
sorry!!